libraries/stumbler/src/main/java/org/mozilla/mozstumbler/service/stumblerthread/scanners/StumblerFilter.java

```diff
@@ -21,7 +22,7 @@
     private static final double MIN_ALTITUDE = -418;      // Dead Sea's altitude in meters
     private static final float MAX_SPEED = 340.29f;   // Mach 1 in meters/second
     private static final float MIN_ACCURACY = 500;       // meter radius
-    private static final long MIN_TIMESTAMP = 946684801; // 2000-01-01 00:00:01
+    private static final long MIN_TIMESTAMP = Build.TIME; // The build time of the application
```

Comment:
This is wrong.  Build.TIME is the buildtime of the operating system, not the application. We don't seem to record the application build time anywhere and Android doesn't do it out of the box for us.